sBTC SIP

```diff
@@ -0,0 +1,166 @@
+# sBTC SIP
+
+## SIP Number: 022
+**Title:** A Decentralized Two-Way Bitcoin Peg  
+**Type:** Standard  
+**Status:** Draft  
+
+## Abstract
+sBTC is a novel digital asset that lets you move Bitcoin in and out of the Stacks blockchain. With sBTC, users can interact with Clarity smart contracts, which enable Bitcoin applications such as payments, decentralized lending, decentralized exchanges, and BTC-backed stablecoins.
+
+sBTC is a SIP-010 token on the Stacks blockchain, backed 1:1 against BTC, and operated by a decentralized set of signers. When BTC is locked on the Bitcoin L1, an equivalent amount of sBTC is issued on the Stacks layer, ensuring a consistent 1:1 ratio of sBTC:BTC. Users can redeem their sBTC at any time by submitting a withdrawal request. Once the request is processed by sBTC signers, BTC is returned to the user’s specified address on the Bitcoin L1.
+
+sBTC will have a crucial role in scaling Bitcoin, as well as introducing new and innovative functionalities to users, growing the size of the overall Bitcoin ecosystem. This SIP aims to describe the sBTC system, the process for signer selection, and the features available in the initial release compared to subsequent versions of sBTC. It does not attempt to describe the low-level technical details of any subsequent release, which will be provided in a future SIP.
+
+## Introduction
+
+## Glossary
+
+| Term                     | Definition |
+|--------------------------|------------|
+| SIP-10 Token             | A token on the Stacks blockchain that adheres to the fungible token standards outlined in SIP-10. |
+| sBTC                     | A SIP-10 token on the Stacks Blockchain that can be turned back into BTC on the Bitcoin Blockchain. 1 sBTC is equivalent to 1 BTC on the Bitcoin Blockchain. |
```

Comment:
Are we sure it's going to be a SIP-010 token?

Reply:
I would argue that since they're referring to this as a SIP-010 token in several places, the expectation is that it will be just that. I may change my opinion after learning more about upcoming SIP this is related to.

Reply:
Is there any reason why it would not be a SIP-010 token?

Reply:
Confirmed, it's a SIP-010 token: 
https://github.com/stacks-network/sbtc/blob/main/contracts/contracts/sbtc-token.clar

Reply:
I was asking in light of the concerns raised [here](https://github.com/stacksgov/sips/pull/176#issuecomment-2192362970) about SIP-010 being inadequate.  FWIW it's totally fine by me if the sBTC SIP proposes to use a different fungible token standard if SIP-010 is too limiting.